db/migrate/20160307085545_warn_about_duplicate_usernames.rb

```diff
@@ -0,0 +1,21 @@
+class WarnAboutDuplicateUsernames < ActiveRecord::Migration
+  def up
+    names = User.group('LOWER(username)').having('count(*) > 1').pluck('LOWER(username)')
+    if names.length > 0
+      puts "-----------------------------------------------------"
+      puts "--------------------- WARNiNG -----------------------"
+      puts "-------- Found users with duplicate usernames -------"
+      puts "-----------------------------------------------------"
+      puts "For the users to log in using their username they have to change it to a unique name"
```

Comment:
Nice!

Maybe "you have to change them in the Rails console"?  Or is there another way?